demisto_sdk/commands/update_release_notes/update_rn.py

```diff
@@ -153,9 +153,10 @@ def is_bump_required(self):
             sys.exit(0)
         return True
 
-    def only_readme_changed(self):
+    def only_docs_changed(self):
         changed_files = self.added_files.union(self.modified_files_in_pack)
-        if len(changed_files) == 1 and 'README' in changed_files.pop():
+        if (len(changed_files) == 1 and 'README' in changed_files.pop()) or \
+                (all('README' in file or '.png' in file for file in changed_files)):
```

Comment:
this is not good as non-doc images might be here...maybe use find_type to check for README/DOC_IMAGE?

Reply:
Tried, but tests are heavily dependent on paths(and not real files)
I'll try something else.